draft-ietf-rats-architecture.md

```diff
@@ -130,43 +124,44 @@ Appraisal Policy for Attestation Result:
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the Evidence it has appraised
 
 Attester:
 
-: An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
+: An entity (typically a device), whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation
 
 Claim:
 : A piece of asserted information, often in the form of a name/value pair.
 (Compare /claim/ in {{RFC7519}})
 
 Endorsement:
 
-: A secure statement that some entity (typically a manufacturer) vouches for the integrity of an Attester's signing capability
+: Statements that Endorsers make (typically a manufacturer) that vouches for the design and implementation of the Attester. Often this includes statements about the integrity of an Attester's signing capability
 
 Endorser:
 
-: An entity that creates Endorsements that can be used to help to appraise the trustworthiness of Attesters
+: An entity (typically a manufacturer) whose Endorsements help Verifiers appraise the authenticity of Evidence
 
 Evidence:
 
-: A set of information about an Attester that is to be appraised by a Verifier
+: A set of information that asserts the trustworthiness status of an Attester, that is appraised by a Verifier
```

Comment:
I believe the new text is less correct than the original.   For example, I would not say that a GPS claims is really about the "trustworthiness status" of an Attester, per se.   It's certainly information about an Attester though.  So I think Evidence can contain some info about the trustworthiness status, as well as additional claims that are not.  (E.g., a public key generated by the target environment that it wants to use for communication with a relying party).   As such, I prefer the original.